src/rpdk/core/project.py

```diff
@@ -434,7 +438,10 @@ def _set_docs_properties(
 
         prop_type = prop.get("type", "object")
 
-        if prop_type in BASIC_TYPE_MAPPINGS:
+        if isinstance(prop_type, list):
+            mapped = MULTIPLE_TYPE_MAPPINGS[MULTIPLE]
+            prop["jsontype"] = prop["yamltype"] = prop["longformtype"] = mapped
```

Comment:
Here is to generate the doc.  We actually could show all the types in the doc, it's different as to generate the Java model.  If we only generate 'Object' for now,  not need the extra `MULTIPLE_TYPE_MAPPINGS[MULTIPLE]`